formula_renames.json

```diff
@@ -2,5 +2,7 @@
   "shiboken"     : "shiboken@1.2",
   "pyside"       : "pyside@1.2",
   "pyside-tools" : "pyside-tools@1.2",
-  "coin"         : "coin@3.1.3"
+  "coin"         : "coin@3.1.3",
+  "qt"           : "qt-legacy-formula",
```

Comment:
No, as that would break everyone who was using QtWebkit. `qt-legacy-formula` installs `qt@4` and `qt-webkit@2.3`, and warns people that they likely need to re-run `cmake`/`qmake` on all the external software they use.